examples/dataframe/delete_table.py

```diff
@@ -0,0 +1,6 @@
+from nisystemlink.clients.dataframe import DataFrameClient
+
+client = DataFrameClient()
+
+# Delete table by id
+client.delete_table("63912927256e5501f9a3cca6")
```

Comment:
I think the API is going to be discoverable enough that maybe we can drop this example.